control-plane/subcommand/inject-connect/command.go

```diff
@@ -476,7 +476,7 @@ func (c *Command) Run(args []string) int {
 	}
 
 	if err := (&gatewaycontrollers.GatewayClassController{
-		ControllerName: gatewaycontrollers.GatewayClassControllerName,
+		ControllerName: gatewaycommon.GatewayClassControllerName,
```

Comment:
ideally we'd probably just use the constant directly within the controller itself rather than doing this, for consistency like above, but this is fine to keep like this for now